glib/src/subclass/object.rs

```diff
@@ -242,6 +283,37 @@ impl<T: ObjectImpl> ObjectImplExt for T {
         }
     }
 
+    #[inline]
+    fn parent_notify(&self, pspec: &ParamSpec) {
+        unsafe {
+            let data = T::type_data();
+            let parent_class = data.as_ref().parent_class() as *mut gobject_ffi::GObjectClass;
+
+            if let Some(ref func) = (*parent_class).notify {
+                func(
+                    self.obj().unsafe_cast_ref::<Object>().to_glib_none().0,
+                    pspec.to_glib_none().0,
+                );
+            }
+        }
+    }
+
+    #[inline]
+    fn parent_dispatch_properties_changed(&self, pspecs: &[ParamSpec]) {
+        unsafe {
+            let data = T::type_data();
+            let parent_class = data.as_ref().parent_class() as *mut gobject_ffi::GObjectClass;
+
+            if let Some(ref func) = (*parent_class).dispatch_properties_changed {
+                func(
+                    self.obj().unsafe_cast_ref::<Object>().to_glib_none().0,
+                    pspecs.len() as _,
+                    pspecs.to_glib_none().0,
```

Comment:
This creates a new temporary allocation for adding a `NULL` at the end, doesn't it? You probably want `pspecs.as_ptr() as *mut _` or so here.